GoogleForCreators/web-stories-wp
# Hotlinking: Allow Link icon to be hotlinked

## Context



## Summary

Allow Link icon to be hotlinked.

## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Enable icon hotlink experiment. 
2. Create a story. 
3. Select page. 
4. Change to style tab.
5. Add url to call to action url input. 
6. After icon option appears, click on pencil icon. 
7. Click on link to file. 
8. Use this URL - https://images.unsplash.com/photo-1656523267493-31b9b2cfdc47?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxlZGl0b3JpYWwtZmVlZHwxfHx8ZW58MHx8fHw%3D&auto=format&fit=crop&w=96&q=60&h=96
9. See icon appear. 

###  Element Icon visible

1. Add element to the page
2. Add a link to the element
3. See the icon image on the panel and on hover of the element.

### Page attachment

1. Add Call to Action to page (https://www.wikipedia.org)
2. See link icon load
3. Note the image logo does pull through beside the Learn More text.

 

Please test other urls, see if they error because of incorrect size. 


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #11777 
Fixes #11847
Fixes #11848
